django_tables2/static/django_tables2/semantic.css

```diff
@@ -0,0 +1,8 @@
+.table-container th.asc:after {
+    content: '\0000a0\0025b2';
+    float: right;
+}
+.table-container th.desc:after {
+    content: '\0000a0\0025bc';
+    float: right;
+}
```

Comment:
I added the CSS following the bootstrap one to see if it was needed. I've dropped it since is not the case.